contracts/Vault.vy

```diff
@@ -1155,7 +1159,7 @@ def expectedReturn(_strategy: address = msg.sender) -> uint256:
 
 
 @external
-def report(_return: uint256) -> uint256:
+def report(_gain: uint256, _loss: uint256) -> uint256:
```

Comment:
Please can we have a second report method with 1 input that calls this with loss = 0. So that this vault upgrade will still work with legacy strategies

Reply:
You can't migrate a strategy to a newer Vault, the solution here will be to upgrade and re-release the strategy.